.github/scripts/update-translations/TranslationManager.py

```diff
@@ -0,0 +1,134 @@
+'''
```

Comment:
(All my naming related comments are just recommendations)

updateTrainslations.py would be a better name for this, I'd probably split it into two, a separate file for createInitialMasterTranslation.py ("developer functions") or something as there is practically no common code for those two functions.